cms/jinja2/templates/pages/statistical_article_page.html

```diff
@@ -189,6 +189,20 @@ <h3 class="ons-corrections-notices__banner-title ons-u-mb-2xs">{{ corrections_an
                 }}
                 {# fmt:on #}
             {% endwith %}
+
+            {% if page.datasets %}
+                {% with title=_("Explore Data"), aria_label=_("Data related to this page"), related_data_text=_("View data used in this article"), related_data_url=routablepageurl(page.specific, "related_data") %}
```

Comment:
```suggestion
                {% with title=_("Explore Data"), aria_label=_("Data related to this page"), related_data_text=_("View data used in this article"), related_data_url=routablepageurl(page, "related_data") %}
```

`page` here is already the specific instance

Reply:
Addressed in [1a7a89b](https://github.com/ONSdigital/dis-wagtail/pull/165/commits/1a7a89be9bd8fb2dd583a492e5330ba3206d691c)